### PR TITLE
[Docs] Refine `community/contributing.md`

### PR DESCRIPTION
**Pull request by TianWen580**
Added the prompt to submit the pull request part and modified a relevant illustration, added a missing command, and modified two missing symbols.
补充了提交拉取请求部分的提示并修改相关配图，补充了一个缺失命令，修改了两处符号缺失

### CODE REVIEW

File: docs/en/community/contributing.md [hunk 1]
```diff
@@ -142,15 +143,15 @@ This will allow you to use the `git push` command to push code directly next tim
 
 <img src="https://user-images.githubusercontent.com/27466624/204302289-d1e54901-8f27-4934-923f-fda800ff9851.png" width="1200">
 
-(2) Modify the PR description according to the guidelines so that other developers can better understand your changes
+(2) Modify the PR description according to the guidelines so that other developers can better understand your changes(note the button *base* on the left side of the PR branch which is needed to modified to dev branch)
 
-<img src="https://user-images.githubusercontent.com/27466624/204303311-84456397-ee41-44f9-945c-85ce415da235.png" width="1200">
+![204303311-84456397-ee41-44f9-945c-85ce415da235](https://i.imgur.com/1CmhbOQ.jpg)
```
Comment (RangeKing) [inline docs/en/community/contributing.md]: Upload images to `githubusercontent` and use the previous format(``)

File: docs/zh_cn/community/contributing.md [hunk 2]
```diff
@@ -144,9 +145,9 @@ git push -u origin {branch_name}
 (1) 在 GitHub 的 Pull request 界面创建拉取请求
 <img src="https://user-images.githubusercontent.com/27466624/204302289-d1e54901-8f27-4934-923f-fda800ff9851.png" width="1200">
 
-(2) 根据指引修改 PR 描述，以便于其他开发者更好地理解你的修改
+(2) 根据指引修改 PR 描述，以便于其他开发者更好地理解你的修改（注意在PR branch左侧的base需要修改为dev分支）
 
-<img src="https://user-images.githubusercontent.com/27466624/204303311-84456397-ee41-44f9-945c-85ce415da235.png" width="1200">
+![204303311-84456397-ee41-44f9-945c-85ce415da235](https://i.imgur.com/OFJSlfC.jpg)
```
Comment (RangeKing) [inline docs/zh_cn/community/contributing.md]: Upload images to `githubusercontent` and use the previous format(``)

File: docs/zh_cn/community/contributing.md [hunk 3]
```diff
@@ -144,9 +145,9 @@ git push -u origin {branch_name}
 (1) 在 GitHub 的 Pull request 界面创建拉取请求
 <img src="https://user-images.githubusercontent.com/27466624/204302289-d1e54901-8f27-4934-923f-fda800ff9851.png" width="1200">
 
-(2) 根据指引修改 PR 描述，以便于其他开发者更好地理解你的修改
+(2) 根据指引修改 PR 描述，以便于其他开发者更好地理解你的修改（注意在PR branch左侧的base需要修改为dev分支）
```
Comment (RangeKing) [inline docs/zh_cn/community/contributing.md:148]: There should be a space between (English characters, numbers) and Chinese characters.

**Review by TianWen580 — COMMENTED** [review]
Unified picture host.

File: docs/en/community/contributing.md [hunk 4]
```diff
@@ -142,15 +143,15 @@ This will allow you to use the `git push` command to push code directly next tim
 
 <img src="https://user-images.githubusercontent.com/27466624/204302289-d1e54901-8f27-4934-923f-fda800ff9851.png" width="1200">
 
-(2) Modify the PR description according to the guidelines so that other developers can better understand your changes
+(2) Modify the PR description according to the guidelines so that other developers can better understand your changes(note the button *base* on the left side of the PR branch which is needed to modified to dev branch)
```
Comment (RangeKing) [inline docs/en/community/contributing.md:146]:
```{note}
The *base* branch should be modified to *dev* branch.
```

**Review by RangeKing — APPROVED** [review]
Please fix the lint issue and then this PR can be merged.

**Pull request closed without merging**